framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -20,6 +20,9 @@
     /** The upper bound of the interval, inclusive. */
     public final long to;
 
+    /** Should this range take overflow into account or ignore it? */
```

Comment:
Since this is a static variable, the documentation about "this range" is misleading.
Should the variable be instance or the documentation be changed?